.github/workflows/openqa.yml

```diff
@@ -18,6 +22,7 @@ env:
 jobs:
   trigger_and_monitor_openqa:
     runs-on: ubuntu-latest
+    environment: manual_approval
```

Comment:
I couldn't read from your commits that this change would be included. Can you please rephrase your git commit message(s) and PR description to explain the reason and the effect of this change?